src/main/resources/db/migration/V6__adding_unique_constraints_dropping_status_add_new_field.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE organisation ADD CONSTRAINT SRA_ID_UQ1 UNIQUE (SRA_ID);
+
+ALTER TABLE professional_user DROP COLUMN status;
+
+ALTER TABLE professional_user ADD COLUMN deleted timestamp;
+
+ALTER TABLE professional_user DROP CONSTRAINT EMAIL_ADDRESS_UQ1;
+
+ALTER TABLE professional_user ADD CONSTRAINT EMAIL_ADDRESS_UQ2 UNIQUE (EMAIL_ADDRESS, DELETED);
```

Comment:
Have you tried re-using the constraint name EMAIL_ADDRESS_UQ1? This would be better if it didn't present any problems.